LODOffers-Licenses-Prices-2020.ttl

```diff
@@ -224,7 +224,7 @@ duration:Duration-2020-10-LOD-Evaluation-30
 
  duration:Duration-2020-12-LOD-Subscription-90
      a opllic:LicenseDuration ;
-     schema:name "Non expiring"^^xsd:string ;
+     schema:name """90 Day Duration """^^xsd:string ;

```

Comment:
Seems to me, this should be the same as the recent `One Year`, i.e., `Duration` is the name/header of the attribute and should not be repeated within its value.
```suggestion
     schema:name """90 Days"""^^xsd:string ;

```

Reply:
I kept it consistent with the 30 day license.

Reply:
I would suggest if you want that to different then either create a new pull request with changes to both the 30 and 90 day licenses or merge this pull request but create new one for the 30 day license.

Reply:
[I see 3 licenses](https://shop.openlinksw.com/shop.vsp?dbms=http%3A%2F%2Fwww.openlinksw.com%2Fontology%2Fsoftware%23WorldWideWeb)

Today, their bullets say "30 Day Duration", "Non expiring", and "365 Days". "90 Day Duration" is viable on *this* page, but "Duration" is redundant on the "License Detail" pages

There are other things I'm noticing, particularly when I review both the "[3 on a page](https://shop.openlinksw.com/shop.vsp?dbms=http%3A%2F%2Fwww.openlinksw.com%2Fontology%2Fsoftware%23WorldWideWeb)" and the individual "License Detail" pages ([30](https://shop.openlinksw.com/offers/order?uri=http%3A%2F%2Fdata.openlinksw.com%2Foplweb%2Foffer%2FOffer-2020-10-LOD-personal-client-access-30-day-sql%23this&type=buy&mode=v), [90](https://shop.openlinksw.com/offers/order?uri=http%3A%2F%2Fdata.openlinksw.com%2Foplweb%2Foffer%2FOffer-2020-10-LOD-personal-client-access-90-day-sql%23this&type=buy&mode=v), [365](https://shop.openlinksw.com/offers/order?uri=http%3A%2F%2Fdata.openlinksw.com%2Foplweb%2Foffer%2FOffer-2020-10-LOD-personal-client-access-365-day-sql%23this&type=buy&mode=v))

Their respective text descriptions say "30 Day Evaluation License", "90 Day Evaluation License", and "365 Day Evaluation License", all "for a Single Network Client License Offer". ("License" occurs twice in each description; I think the first occurence should become "License Offer" and the trailing "License Offer" should be deleted.)

I think the (free) "30 Day Evaluation License" should retain "Evaluation", while the other two should lose "Evaluation".

@kidehen @openlink — Please review and merge this PR without my suggested changes, and I'll make a new PR for all the changes I am seeing as necessary.